pkg/lvm/lvm.go

```diff
@@ -859,8 +859,14 @@ func run(cmd string, v interface{}, extraArgs ...string) error {
 		// We use Lock instead of TryLock as we have no alternative way of
 		// making progress. We expect lvm2 command-line utilities invoked by
 		// this package to return within a reasonable amount of time.
-		lvmlock.Lock()
-		defer lvmlock.Unlock()
+		if lerr := lvmlock.Lock(); lerr != nil {
+			return lerr
```

Comment:
Looking at the impl of flock the only possible errors we might get would be related to failures using the configured file. Does it make sense to decorate this error with that additional context so users see e.g., `Could not lock configured file lock: ...`?